tensorflow/lite/python/interpreter.py

```diff
@@ -183,7 +183,8 @@ class Interpreter(object):
   def __init__(self,
                model_path=None,
                model_content=None,
-               experimental_delegates=None):
+               experimental_delegates=None,
+               num_threads=1):
```

Comment:
Let's keep the default None, so it remains implementation-dependent, and we skip setting the threads if it's None.

Reply:
done